### DISCUSSION TITLE
The quality of the time series degrades as the distance from the reference point increases.

### DISCUSSION THREAD

**Discussion by m1ch96** [insarlab/MintPy#1166] [Q&A]
Dear developers of Mintpy

I'm trying to perform a time series analysis of my stack of 1771 interferograms processed with GMTSAR.
I have followed the two examples of workflows in Mintpy:
https://github.com/insarlab/MintPy-tutorial/blob/main/workflows/smallbaselineApp_aria.ipynb
https://github.com/insarlab/MintPy-tutorial/blob/main/workflows/smallbaselineApp.ipynb
And I loaded the data following these instructions:
https://mintpy.readthedocs.io/en/latest/dir_structure/

I encountered no errors apart from the unwrap error correction (which I skipped in the end, as it is optional), and I obtained the time series of displacement along the line of sight. However, the quality of the time series obtained is very low, and they appear very scattered. The main point that I noticed is that the further the time series associated with a pixel is from the reference point pixel, the worse the quality. As an example, I attach three images with increasing distances from the reference point (the first is near, etc.). I corrected the troposphere with the ERA5 dataset and the DEM error.
![Point_Displacement_Time-series_near1](https://github.com/insarlab/MintPy/assets/128250524/469c9de7-6eba-4cbb-a173-3f18c9d78cdf)
![Point_Displacement_Time-series_medium](https://github.com/insarlab/MintPy/assets/128250524/790c096c-91f5-4900-826a-aee2a9f0553d)
![Point_Displacement_Time-series_far](https://github.com/insarlab/MintPy/assets/128250524/442cfa24-d9d9-4a09-8a25-91f72b829ca9)

This frame is characterized by a low coherence on average (~0.10). However, I observed some other processing of this frame, and the quality of the time series obtained is not as low as mine.
[original_averagecoh.pdf](https://github.com/insarlab/MintPy/files/14724577/original_averagecoh.pdf)

I also attach some other graphs that characterize this specific run.
[0.10_0.15_intfpersar.pdf](https://github.com/insarlab/MintPy/files/14724628/0.10_0.15_intfpersar.pdf)
[0.10_0.15_network.pdf](https://github.com/insarlab/MintPy/files/14724629/0.10_0.15_network.pdf)
[0.10_0.15_coherenceHistory.pdf](https://github.com/insarlab/MintPy/files/14724630/0.10_0.15_coherenceHistory.pdf)
[0.10_0.15_pbaseHistory.pdf](https://github.com/insarlab/MintPy/files/14724631/0.10_0.15_pbaseHistory.pdf)
[0.10_0.15_avgSpatialCoh.pdf](https://github.com/insarlab/MintPy/files/14724632/0.10_0.15_avgSpatialCoh.pdf)
[temporalCoherence.pdf](https://github.com/insarlab/MintPy/files/14724790/temporalCoherence.pdf)

I tried to choose the reference point based on these criteria:
1. Low deformation pixel
2. High coherence
3. Not affected by strong ionospheric streaks
4. Close to the same elevation as the area I am studying
The last point is difficult for me since my AOI covers the entire frame.

I tried several combinations of minCoherence, minAreaRatio, and reference point positions but the pattern I observe in the end is always the same: a degradation of the quality of the time series farther from the reference point position.

At the bottom, I attached my custom template. At the beginning, there are parameters that characterize my workflow and the workflow of GMTSAR.

I tried to look in the forum and here, but I was not able to find someone with a similar problem.  Could the issue be attributed to the absence of the unwrap error correction phase? If you have any idea what might be causing the issue or if you need further information, please let me know. 

Thanks for your attention.
Michele

**System information**
* Operating system: CentOS Linux release 7.9.2009 kernel: 3.10.0-1160.25.1.el7.x86_64
* Python environment: Conda
* MintPy version: MintPy version 1.5.3, date 2023-11-23
* InSAR processor/product: gmtsar
* Your custom / default template file:  [conffile.txt](https://github.com/insarlab/MintPy/files/14725140/conffile.txt)

**Comment by welcome** [comment]
👋  Thanks for opening your first issue here! Please filled out the template with as much details as possible. We appreciate that you took the time to contribute!
Make sure you read our [contributing guidelines](https://github.com/insarlab/MintPy/blob/master/docs/CONTRIBUTING.md).

**Comment by yunjunz** [comment]
Hi Michele,

There are usually two reasons: 1) atmospheric delays, which are spatially correlated and, thus, are larger as the distance increases; 2) phase unwrapping errors. And in your case, it's both. The atmospheric delay is a natural noise source, and there is little to do besides applying the currently available methods. PU errors are processing errors and could be further mitigated to some extent. 

The PU errors here are mainly due to the sparse distribution of the coherent urban areas. If high-resolution urban deformation is your signal of interest, you should be better off using the PS approach, as available in isce2/snap + stamps workflow, or isce2 + miaplpy + mintpy workflow. 

The gmtsar + mintpy workflow uses the small baseline approach, which is better for low-resolution large spatial wavelength deformation, if I understand it right. If you want to continue with this workflow, I would recommend the following:
+ tune the network modification to drop the low coherent interferograms. Based on the network.pdf file, no interferograms are excluded yet.
+ if the above change is not enough (PU error exists, which is shown as low temporal coherence value), split the area into multiple smaller ones, and apply the mintpy analysis to each of them independently, to limit the distance to the reference point.
+ if the above two changes are not enough, then re-run gmtsar processing with more looks and stronger filter strength, to increase the interferogram coherence.

Yunjun
